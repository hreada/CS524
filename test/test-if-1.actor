if(eq?(0,0),1,0)
