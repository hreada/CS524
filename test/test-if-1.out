<T>
  <actors>
    <actor>
      <k> exec ( 1 ) </k>
      <id> initactor </id>
    </actor>
  </actors>
  <messages> .MessageCellBag </messages>
  <definedaddr> ListItem ( initactor ) </definedaddr>
</T>
